From acf183c4ad301a1db16c91b9db3b547341dad90a Mon Sep 17 00:00:00 2001
From: Tommy Huang <tommy_huang@aspeedtech.com>
Date: Fri, 19 Sep 2025 11:30:23 +0800
Subject: [PATCH 09/10] dts: ast2700: add mail box to bootmcu setting

Add mail box to bootmcu setting for ltpi access
usage.

Signed-off-by: Tommy Huang <tommy_huang@aspeedtech.com>
Change-Id: I1b991c42380c1aebc2bb87e07319e04d3df2abea
(cherry picked from commit 7b792413d9cff55a9b806932da8cfbb6606de3ce)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 arch/arm/dts/ast2700-evb.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/dts/ast2700-evb.dts b/arch/arm/dts/ast2700-evb.dts
index 524536dc2b..4d5a63303c 100644
--- a/arch/arm/dts/ast2700-evb.dts
+++ b/arch/arm/dts/ast2700-evb.dts
@@ -303,6 +303,14 @@
 		mboxes = <&mbox0 0>;
 	};
 };
+
+&soc1 {
+	mbox-bootmcu-0 {
+		compatible = "aspeed,aspeed-mbox";
+		reg = <0x4 0x31a80000 0x0 0x80000>, <0x4 0x31a88000 0x0 0x80000>;
+		mboxes = <&mbox2 0>;
+	};
+};
 #endif
 
 &soc0 {
-- 
2.34.1

